Cubical/Data/Int/IsEven.agda

```diff
@@ -0,0 +1,147 @@
+{-# OPTIONS --safe #-}
+module Cubical.Data.Int.IsEven where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Relation.Nullary
+
+open import Cubical.Data.Empty as ⊥
+open import Cubical.Data.Bool
+open import Cubical.Data.Nat
+  using (ℕ ; zero ;  suc ; +-zero ; +-suc ; injSuc ; +-comm ;snotz ; znots ; ·-distribˡ)
+  renaming
+  (_+_    to _+ℕ_ ;
+   _·_    to _·ℕ_ ;
+   isEven to isEvenℕ ;
+   isOdd  to isOddℕ)
+open import Cubical.Data.Nat.IsEven
+  using ()
+  renaming
+  (isEvenFalse  to isEvenℕFalse   ;
+   isEvenTrue   to isEvenℕTrue    ;
+   isOddFalse   to isOddℕFalse    ;
+   isOddTrue    to isOddℕTrue     ;
+   falseIsEven  to falseIsEvenℕ   ;
+   trueIsEven   to trueIsEvenℕ    ;
+   ¬IsEvenFalse to ¬IsEvenℕFalse  ;
+   ¬IsEvenTrue  to ¬IsEvenℕTrue   )
```

Comment:
Hi, just a comment: I think instead of these renamings it might be better to write `import Cubical.Data.Nat.IsEven as ℕ`, without the `open`, to avoid name clashes.

Reply:
Would this work ? 
From what I remember, importing a module as something doesn't hide the definition without N.X , right ? 
IBut it is still possible to call X directly and so it would conflict with the other definitions ?

Reply:
Yes, it works, I just double-checked, see here: https://github.com/thomas-lamiaux/cubical/pull/1

It is important that you drop the "open". :-)

Reply:
ha yeah, nice, I'll modify that before merging

Reply:
resolved